ruby_02-web_applications_with_ruby/outlines/route_helpers.md

```diff
@@ -9,66 +9,87 @@ tags: routes, helpers
 
 By the end of this lesson, you will know/be able to:
 
-  * Run a command that can find all the information about your routes - know what 5 pieces of information this tool gives us.
-  * Use a link helper to make writing html links easier.
+  * Run `rake routes` to find information about your routes.
+  * Understand the 5 pieces of information this `rake routes` gives us.
   * Use a route helper to easily refer to a relative and absolute path.
+  * Understand the difference between what `_url` and `_path` return when combined with a routes prefix.
+  * Use a link helper to make writing html links easier.
   * Find a routes prefix and use that prefix to build a helper.
 
-### What is CRUD again?
+### Review
+
+What is CRUD again?
 
   * Create
   * Read
   * Update
   * Delete
 
-### What are the specific CRUD routes I get:
+### Warmup
 
-  * If I am modeling the entity `:song`?
-  * If I am modeling the entity `:phone`?
-  * If I am modeling the entity `:product`?
-  * If I am modeling the entity `:sticker`?
+In pairs, one of you write the CRUD routes for `song`, one for `phone`. If you're in a group of three, have the third person write the routes for `sticker`.
 
-##### Do you notice any repetition or patterns here?
+With your partner, do you notice any repetition or patterns in the routes that you wrote?
```

Comment:
This is something they should absolutely be able to answer given that they have already seen the http verb + path name + action table and we've been reiterating it. 

Is there an additional question (or can we re-phrase this one) that's both review but might stretch them a little?